docs/decisiones/gestor-tareas.md

```diff
@@ -0,0 +1,11 @@
+# Elección del gestor de tareas
+
+Existen, entre otros muchos, los siguientes gestores de tareas:
```

Comment:
Los criterios tienes que ponerlos también aquí.

Reply:
Al final del archivo he puesto los criterios que he seguido para elegir el gestor de tareas. Los pongo encima también entonces.

Reply:
Creo que no entendéis que no se trata de escribir las cosas aquí o allí. Se trata de exponer el proceso de toma de decisiones. Si lo pones "al final del archivo" claramente no has seguido ningún criterio en la decisión, sino que te has limitado a escoger algo y luego a escribir cosas para justificarlo. Ya está claro que es así, pero por lo menos trata de respetar formalmente lo que se pide en este objetivo, que es que *primero* entendáis de qué se trata algo y busquéis unos criterios para escogerlo objetivamente y *después* toméis la decisión.